external/js/kyber/lib/util/index.js

```diff
@@ -0,0 +1,35 @@
+"use strict";
+
+/**
+ * Class PRNG defines a PRNG using a Linear Congruent Generator
+ * Javascript doesn't have a seedable PRNG in stdlib and this
+ * implementation is to be used for deterministic outputs in tests
```

Comment:
This is a dangerous thing to include in a crypto library. I understand it is only for test, so then it needs to NOT be in the main library.

Reply:
right, moving this into `test/`